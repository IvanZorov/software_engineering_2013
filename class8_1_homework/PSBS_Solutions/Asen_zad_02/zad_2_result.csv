Pesho,10
Gosho,14
Asen,24
