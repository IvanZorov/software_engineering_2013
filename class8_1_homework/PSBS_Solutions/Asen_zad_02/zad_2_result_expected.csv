as,12
asa,45
as,1
